### DISCUSSION TITLE
Connectivity

### DISCUSSION THREAD

**Discussion by Ecmbuster** [webspiderteam/Bluetooth-DMM-For-Windows#52] [General]
There are a few questions with the application.
I believe W7 is not supported (stalls and locks up) and testing with W10. 
The questions are about the type of Bluetooth device attached to a laptop. 
Bluetooth USB and on-board Bluetooth modules do not recognize the ANENG AN9002 via W10.
 Is there a special requirement for Bluetooth (version) and what version of Windows is required to operate the application.
Best regards

**Comment by webspiderteam** [comment]
Sorry as you said win7 not supported because main libraries uses .net framework 4.5 minimum. Win7 supports only up to 3.5... minimum win8 support 4.5.

Old devices mostly has Bluetooth version lover than 4. As I remember Bluetooth version 3 and over has Bluetooth Low Energy (BLE). These multimeters communicate over BLE GATT protocol. So if you want to use your 9002 with your computer you need a Bluetooth version 4.0. for my app win 10 recommended.

**Comment by Ecmbuster** [comment]
Thanks for the advice and definitely works with Windows 10 using the tp-link 4.0 dongle.
The application is amazing and will discuss it with other users. It also makes a great teaching tool.
The giggle factor has increased X 10. Well done.
Best regards

**Reply by webspiderteam** [reply]
Thanks for sharing your nice thoughts :) I wish app can help much more people :)
